NEWS.md

```diff
@@ -51,7 +51,7 @@ an error (#311).
 
 # tiltIndicator 0.0.0.9057
 
-* `pstr_prepare_scenarios()` now handles "weo" data correctly (@kalashsinghal #309).
+* `pstr_prepare_scenario()` now handles "weo" data correctly (@kalashsinghal #309).
```

Comment:
I'm not updating this because it refers to a point in the history when the function had the old name. However note the name was wrong so I fixed it.